src/modules/battery.cpp

```diff
@@ -109,29 +109,7 @@ const std::string waybar::modules::Battery::getAdapterStatus(uint8_t capacity) c
   return "Unknown";
 }
 
-const std::string waybar::modules::Battery::getState(uint8_t capacity) const {
-  // Get current state
-  std::vector<std::pair<std::string, uint8_t>> states;
-  if (config_["states"].isObject()) {
-    for (auto it = config_["states"].begin(); it != config_["states"].end(); ++it) {
-      if (it->isUInt() && it.key().isString()) {
-        states.emplace_back(it.key().asString(), it->asUInt());
-      }
-    }
-  }
-  // Sort states
-  std::sort(states.begin(), states.end(), [](auto& a, auto& b) { return a.second < b.second; });
-  std::string valid_state;
-  for (auto const& state : states) {
-    if (capacity <= state.second && valid_state.empty()) {
-      label_.get_style_context()->add_class(state.first);
-      valid_state = state.first;
-    } else {
-      label_.get_style_context()->remove_class(state.first);
-    }
-  }
-  return valid_state;
-}
+
```

Comment:
Same

Reply:
oh did you mean the now redundant method here? i deleted the space but you probably meant the extra getState implementation?

Reply:
Nope that's it